language-design.md

```diff
@@ -59,6 +100,13 @@ What does the post on grayscale tell us about the process of API design?
 
 **Response**
 
+First and foremost, the post and its comments show us that it’s extremely important to ask for feedback during the design process.  Almost every comment provided valuable feedback, either pointing out pros and cons of the suggested names or suggesting new names entirely.  The commenters were generally in agreement that they wanted the name that felt most intuitive (7 separate commenters even used the word “intuitive” in their posts).  However, they could not seem to agree on what that name was.  Out of the 756 respondents of the [poll]( https://docs.google.com/forms/d/1pp3RY-A4MAs7b-gmqFx6bKn52_G_WLoPFkV0vueiWP4/viewanalytics?usp=form_confirm), at least 9% voted for each of the five options, meaning that for every possible name, a sizable portion of users found it to be the most intuitive.  More importantly, the most popular choice, “rgb(x) and rgba(x, a),” received only ~47% of the vote, meaning that at least half of the users would not be fully satisfied no matter what name the developers choose.  A takeaway from this, I think, is that when designing an API you may come across situations where it’s impossible to please everyone.  When each possible name has a legitimate list of pros and cons, there won’t be one right solution.  Accepting this fact and not pandering to vocal minorities is important.
+
+
+One commenter put forth the print/photography terms “luma, lightness, value, brightness, intensity, or key” as possible names, and implied that users with backgrounds in print or photography might appreciate the constancy.  This brings up an interesting point, which is that the intuitiveness of a name will vary greatly depending on the user’s background.  This would help to explain the mixed poll results, and reinforces the fact that, as a developer of an API, it is important to always keep in mind the backgrounds of your users.
+
+
+The mixed responses bring to mind the advice of Joshua Bloch, who suggests that “if it’s hard to find good names, go back to the drawing board” [Bloch 2006].  One comment was in line with this advice and asked the developers, “Why the need for this function at all? Seems unnecessary.”
```

Comment:
I was actually pretty confused about that quote from Bloch when I read it. I wasn't sure if by "go back to the drawing board" he meant with the entire API, with the API's organization, or with just the names of the functions. In this case, it seems like this is such a small scale change that going back to the drawing board means rethinking if its even necessary, but for a larger API implementation it seems like how far you should go back is a hard question.